cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -209,6 +211,38 @@ func startControllers(c *cloudcontrollerconfig.CompletedConfig, rootClientBuilde
 	versionedClient := rootClientBuilder.ClientOrDie("shared-informers")
 	sharedInformers := informers.NewSharedInformerFactory(versionedClient, resyncPeriod(c)())
 
+	var clusterCIDR, serviceCIDR *net.IPNet
+	if c.Generic.ComponentConfig.AllocateNodeCIDRs {
```

Comment:
This should be 
```
if c.Generic.ComponentConfig.KubeCloudShared.AllocateNodeCIDRs {
}
```
Wonder how this complies at all 🤔 unless we don't compile this binary at all in CI which we should fix

Reply:
Verified
```
cmd/cloud-controller-manager/app/controllermanager.go:212:30: c.completedConfig.Generic.completedConfig.Config.ComponentConfig.AllocateNodeCIDRs undefined (type componentconfig.KubeControllerManagerConfiguration has no field or method AllocateNodeCIDRs)
```